cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/GetLogic.java

```diff
@@ -53,7 +53,7 @@ public ConfigBean getAll(SpaceDevelopersOperations spaceDevelopersOperations,
         ConfigBean configBean = new ConfigBean();
         SpecBean specBean = new SpecBean();
         // start async querying of config data from the cloud foundry instance
-        log.debug("Start async querying of apps, services and space developers...");
+        log.debug("Querying apps, services and space developers asynchronously");
```

Comment:
I think we can remove the 'asynchronously' info. I added it when it was changed from sequential. But now almost all requests can work asynchronously. So seems kinda odd.

Reply:
True. Let me fix that.